src/reducers/linter.spec.tsx

```diff
@@ -378,6 +379,7 @@ describe(__filename, () => {
       expect(
         createInternalMessage({
           ...fakeExternalLinterMessage,
+          id,
```

Comment:
Could you also fill in some value for `id` in the [helper](https://github.com/mozilla/addons-code-manager/blob/9457fa8069298c76d0f32dbe288dc7f257b2ea1e/src/test-helpers.tsx#L128)?

It looks like pretty much everything will have a code: https://github.com/mozilla/addons-linter/blob/master/src/messages/javascript.js